plugins/source/azuredevops/codegen/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"log"
+
+	"github.com/cloudquery/cloudquery/plugins/source/azuredevops/codegen/recipes"
+)
+
+func main() {
+	var resources []*recipes.Resource
+	resources = append(resources, recipes.Core()...)
+
+	for _, resource := range resources {
+		if err := resource.Generate(); err != nil {
```

Comment:
use errgroup, like this:
```go
grp, _ := errgroup.WithCOntext(context.Background())
for _, resource := range resources {
	grp.Go(resource.Generate)
}
if err := grp.Wait(); err != nil {
	log.Fatal(err)
}
```

Reply:
What is the impact of adding this code now? We only have 1 resource.

The purpose of the PR is to have basic code gen for AzureDevOps. I'd rather do any optimizations in follow up PRs to keep the PR small.

Reply:
Yes, there is no need to add goroutines here.